charts/consul/values.yaml

```
@@ -205,6 +206,9 @@ global:
       # Please refer to [Vault ACL policies](https://developer.hashicorp.com/consul/docs/connect/ca/vault#vault-acl-policies)
       # documentation for information on how to configure the Vault policies.
       connectCA:
+        # Vault namespace
```

~ Align w/ other descriptions (if we keep this config key here)
```suggestion
        # Vault namespace (optional).
        # For more details, please refer to [Vault Connect CA configuration](https://developer.hashicorp.com/consul/docs/connect/ca/vault#namespace)
```